scripts/rebase.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+set -ex
+
+base_brach=${1:-master}
+head_branch=${2:-next}
+
+git fetch origin "$base_brach"
+git fetch origin "$head_branch"
+git checkout -b "$head_branch" "origin/$head_branch"
+
+is_rebasing=1
+
+exitcode=0 && git rebase "origin/$base_brach" || exitcode=$?
+if [[ $exitcode -eq 0 ]]; then is_rebasing=0; fi
+
+while [[ $is_rebasing == 1 ]]; do
+  conflicting_files="$(git diff --name-only --diff-filter=U)"
+  fixable_files="$(git diff --name-only --diff-filter=U yarn.lock tests/snapshots)"
+  if [[ "$conflicting_files" == "$fixable_files" ]]; then
+    yarn install
+    yarn test -u
+    git add yarn.lock tests/snapshots
+    exitcode=0 && git rebase --continue || exitcode=$?
+    if [[ $exitcode -eq 0 ]]; then
+      is_rebasing=0
+      break
+    fi
+  else
+    break
```

Comment:
why bother to break at the very end of the loop?

Reply:
Because `$is_rebasing` might still be set to `1` but then it is a conflict that can't be resolved automatically, therefore I have to break out of the loop.